tokio-stream/src/stream_ext/chunks_timeout.rs

```diff
@@ -67,7 +67,7 @@ impl<S: Stream> Stream for ChunksTimeout<S> {
         }
 
         if !me.items.is_empty() {
-            ready!(me.deadline.poll(cx));
+            ready!(me.deadline.as_pin_mut().unwrap().poll(cx));
```

Comment:
I don't like this - it's too fragile. You should check for `None` and not poll instead of using unwrap.
```suggestion
            if let Some(deadline) = me.deadline.as_pin_mut() {
                ready!(deadline.poll(cx));
            }
```
I know that the unwrap can never be triggered as the code is currently written, but in the past, I have made changes to similar code that resulted in a changelog that looks like this:

> ### 1.19.2 (June 6, 2022)
>
> This release fixes another bug in `Notified::enable`. (https://github.com/tokio-rs/tokio/pull/4751)
>
> ### 1.19.1 (June 5, 2022)
>
> This release fixes a bug in `Notified::enable`. (https://github.com/tokio-rs/tokio/pull/4747)